pilot/pkg/model/context.go

```diff
@@ -171,6 +171,14 @@ func (pversion *IstioVersion) Compare(inv *IstioVersion) int {
 	return 0
 }
 
+// ToString converts IstioVersion to a semver format string.
+func (pversion *IstioVersion) ToString() string {
```

Comment:
If this is mixer-specific - I would prefer that it is in mixer package.

Reply:
But not a huge deal - just worried that when mixerv2 and wasm show up we'll be left with dead stuff in pilot.

Reply:
Moved to mixer plugin.